lib/ansible/galaxy/collection/concrete_artifact_manager.py

```diff
@@ -382,7 +384,7 @@ def _extract_collection_from_git(repo_url, coll_ver, b_path):
 
 
 # FIXME: use random subdirs while preserving the file names
-def _download_file(url, b_path, expected_hash, validate_certs, token=None):
+def _download_file(url, b_path, expected_hash, validate_certs, token=None, timeout=60):
     # type: (str, bytes, Optional[str], bool, GalaxyToken) -> bytes
```

Comment:
```suggestion
    # type: (str, bytes, Optional[str], bool, GalaxyToken, int) -> bytes
```

This fix was already applied in devel: https://github.com/ansible/ansible/pull/77147